app/src/userpages/components/ProfilePage/profilePage.pcss

```diff
@@ -3,6 +3,25 @@
 }
 
 .longText {
-  line-height: 24px;
-  max-width: 488px;
+  line-height: 20px;
```

Comment:
`line-height` is specified to be 24px in the designs. This way it matches the stream editor also.